client/me/purchases/purchases-site/reconnect-notice.jsx

```diff
@@ -13,34 +13,35 @@ import React, { Component } from 'react';
  */
 import Notice from 'components/notice';
 import NoticeAction from 'components/notice/notice-action';
-import { CALYPSO_CONTACT, JETPACK_CONTACT_SUPPORT } from 'lib/url/support';
+import { CALYPSO_CONTACT } from 'lib/url/support';
 
 class PurchaseReconnectNotice extends Component {
 	static propTypes = {
 		name: PropTypes.string,
-		domain: PropTypes.string,
+		translate: PropTypes.func.isRequired,
 	};
 
 	render() {
 		const { translate, name, isJetpack } = this.props;
-		let text = translate( 'You are no longer a user on %(site)s and cannot manage this purchase.', {
-			args: {
-				site: name,
-			},
-		} );
-		if ( isJetpack ) {
-			text = translate( '%(site)s has been disconnected from WordPress.com.', {
-				args: {
-					site: name,
-				},
-			} );
-		}
+
+		const text = isJetpack
```

Comment:
Not sure why, but formatting here looks broken - too many indentation tabs?

Reply:
🤷‍♂️ Seems fine to me, it matches indentation in this block (`const` above). The ternary following is more indented.

Either way this is `prettier` at work 💅